requirements/doc.txt

```diff
@@ -16,7 +16,7 @@ attrs==21.4.0
     #   pytest
 babel==2.9.1
     # via sphinx
-bleach==4.1.0
+bleach==5.0.0
```

Comment:
`bleach==5.0.0` contains breaking changes related to `clean()` function used to sanitize css .  Since tests are passing so it can be upgraded.